pandas/tests/extension/decimal/test_decimal.py

```diff
@@ -146,9 +146,10 @@ class TestMissing(BaseDecimal, base.BaseMissingTests):
 
 class Reduce:
     def check_reduce(self, s, op_name, skipna):
-
         if op_name in ["median", "skew", "kurt"]:
-            with pytest.raises(NotImplementedError):
+            # breakpoint()
```

Comment:
Can you remove this comment and put back the blank line there was before under `def check_reduce(self, s, op_name, skipna):`?